parking/shared/util.py

```diff
@@ -24,11 +23,17 @@ def serialize_model(model: object) -> str:
     return json.dumps(attr.asdict(model))
 
 
-def validate_pos(cls, attribute, value: Union[int, float]) -> None:
```

Comment:
Little known fact is Python has a numeric type hierarchy for type hints so type checkers are required to allow int arguments to a float typed parameter so `Union[int, float]` is actually equivalent to `float`.